### PR TITLE
Update jib-cli readme

### PR DESCRIPTION
**Pull request by loosebazooka**
Adds a first readme for the new jib cli

### CODE REVIEW

File: jib-cli/README.md [hunk 1]
```diff
@@ -14,44 +14,232 @@ are almost certain to change.
...
-## Examples
+## Usage
 
-### Using nginx to serve a static website
+Currently only one command is supported: `build`
 
-The following example creates an nginx-based container to serve static content that is found in `path/to/website`.
-The result is loaded to the local Docker daemon as `my-static-website`:
+```
+jib build -t gcr.io/my-project/my-image [options]
```
Comment (chanseokoh) [inline jib-cli/README.md:28]: How about `--target` instead of `-t`? Could be helpful for first time users.

**Review by chanseokoh — COMMENTED** [review]
LGTM

File: jib-cli/README.md [hunk 2]
```diff
@@ -14,44 +14,232 @@ are almost certain to change.
 Use the `application` plugin's `installDist` task to create a runnable installation in
```
[the remark this answers is not in the document]
Comment (loosebazooka) [reply jib-cli/README.md:14]: Oh no that's fine, I'll add it in here.

[the remark this answers is not in the document]
Comment (loosebazooka) [reply jib-cli/README.md:14]: oh I see it's not available yet. Okay I'll leave a TODO in the readme.

**Review by chanseokoh — APPROVED** [review]

**Pull request merged**